packages/ckeditor5-table/src/tablecolumnresize/utils.js

```diff
@@ -365,3 +365,21 @@ export function removeColumnResizerElements( viewWriter, viewCell ) {
 
 	viewWriter.remove( viewTableColumnResizerElement );
 }
+
+// Shows/hides resizers in specified view.
+//
+// @param {module:ui/view~View} view
+// @param {true|false} value
+export function setResizersVisibility( view, value ) {
```

Comment:
`setResizersVisibility()` sounds like it would show the resizer if it's set to `true` which is not entirely true. If it's set to true the resizers will be available but doesn't necessary have to be shown.

I think like `hideAllColumnResizers()` / `restoreAllColumnResizers()` would be a more fitting. But at the same time I'd prefer going with a single function like you did but I don't have good name for that 😄

Reply:
I prefer a single function so I changed its name to `setResizersEnablement`, WDYT?